docs/Learning-Environment-Design-Agents.md

```diff
@@ -878,14 +878,13 @@ will be recorded from the agent.
 <p align="center">
   <img src="images/demo_component.png"
        alt="Demonstration Recorder"
-       width="450" border="10" />
+       width="650" border="10" />
 </p>
 
 When `Record` is checked, a demonstration will be created whenever the scene is
 played from the Editor. Depending on the complexity of the task, anywhere from a
 few minutes or a few hours of demonstration data may be necessary to be useful
-for imitation learning. When you have recorded enough data, end the Editor play
-session. A `.demo` file will be created in the `Assets/Demonstrations` folder
+for imitation learning. To specify an exact number of steps you want to record use the `Num Steps To Record` field and the editor will end your play session automatically once that many steps are recorded. If you set `Num Steps To Record` to `0` then recording will continue until you manually end the play session. Once the play session ends a `.demo` file will be created in the `Assets/Demonstrations` folder
```

Comment:
Maybe break it into several lines of the same length as others?

Reply:
Thanks, done.